pkg/storage/store.go

```diff
@@ -432,7 +432,8 @@ type Store struct {
 
```

Comment:
Copy pasta of the changes to allow empty snapshots to proceed even when a big snapshot is in progress.